benchmarks/conftest.py

```diff
@@ -0,0 +1,177 @@
+import os
+from datetime import datetime
+from decimal import Decimal as PyDecimal
+import pytest
+from openpyxl import load_workbook
+from stock_indicators.indicators.common import Quote
+
+dir = os.path.dirname(__file__)
+data_path = os.path.join(dir, "History.xlsx")
+wb = load_workbook(data_path, data_only=True)
+
+@pytest.fixture(scope='session')
```

Comment:
One thing to check in your implementation is that the loading of historical quotes should not be in the performance test measurement; we should only be testing (measuring) the actual indicator calculation time, after quotes have been fetched.

Reply:
As I know, we're doing so. but I'll have a double-check.